### ISSUE TITLE
desktop notifications show raw HTML code, not plain text

### ISSUE THREAD

**Issue by mauke** [Nheko-Reborn/nheko#1264]
### Describe the bug

When someone mentions me, nheko generates a desktop notification. However, the notification message shows a lot of HTML code and not the plain text content.

### To Reproduce

1. Have nheko running in the background
2. Wait for someone to mention your name
3. Look at desktop notification message

### What happened?

The message shows
```
kim: <a href="https://matrix.to/#/@mauke:matrix.org">mauke</a...
```

### Expected behavior

The message should show
```
<kim> mauke: oh, I see
```

### Version

0.10.2-3a6a9054

### Operating system

Linux

### Installation method

Flatpak nightly repo or download

### Desktop Environment

GNOME

### Did you use profiles?

- [ ] Profiles used?

**Comment by deepbluev7** [comment]
The desktop environment seems to claim it supports rich text then, but not actually do that or the html is in the plain text message.

Can you show the output of either

```
gdbus call -e -d org.freedesktop.Notifications -o /org/freedesktop/Notifications -m org.freedesktop.Notifications.GetCapabilitie
```

or

```
qdbus org.freedesktop.Notifications /org/freedesktop/Notifications org.freedesktop.Notifications.GetCapabilities
```

?

**Comment by mauke** [comment]
```
$ gdbus call -e -d org.freedesktop.Notifications -o /org/freedesktop/Notifications -m org.freedesktop.Notifications.GetCapabilities
(['actions', 'body', 'body-markup', 'icon-static', 'persistence', 'sound'],)
```

**Comment by deepbluev7** [comment]
Well, it claims to support markup. That includes links: https://specifications.freedesktop.org/notification-spec/notification-spec-latest.html#markup

So that doesn't sound like our fault.

**Comment by mauke** [comment]
For reference: `"body-hyperlinks"` is not in the list of returned capabilities, so hyperlinks are not supported by GNOME.

General formatting with `<b>`, `<i>`, `<u>`, etc. works, but `<a>` is not rendered at all. The latter could be considered a bug in GNOME as the specification says "Notification servers that do not support these tags should filter them out" and "The tags specified above mark up the content in a way that allows them to be stripped out on some implementations", which GNOME is clearly not doing.

As a workaround for this issue, could I request that nheko not generate `<a>` tags if the notification server does not support `body-hyperlinks`?

**Comment by deepbluev7** [comment]
We don't really generate those links though. We would need to implement workarounds for specific implementations, which I think would be better to just file an issue with gnome, that they filter them out then, would it not?

**Comment by mauke** [comment]
Makes sense. Do you happen to know where/how I can report this bug upstsream?

**Comment by deepbluev7** [comment]
I think you just use their bugzilla? https://help.gnome.org/users/evolution/stable/problems-reporting-bugs.html.en

http://bugzilla.gnome.org/
